crates/cybercmd/src/config/logging.rs

```diff
@@ -0,0 +1,51 @@
+use std::path::Path;
+
+use anyhow::Result;
+use log::LevelFilter;
+use log4rs::{
+    append::rolling_file::{
+        policy::compound::{
+            roll::fixed_window::FixedWindowRoller, trigger::size::SizeTrigger, CompoundPolicy,
+        },
+        RollingFileAppender,
+    },
+    config::{Appender, Root},
+    encode::pattern::PatternEncoder,
+    Config,
+};
+
+use crate::config::app_context::Paths;
+
+pub(super) fn setup_logging(paths: &Paths) -> Result<()> {
+    let window_roller = FixedWindowRoller::builder()
+        .build(
+            Path::join(Path::new(&paths.logs), "cybercmd{}.log")
```

Comment:
maybe just use `.` syntax
```suggestion
            Path::new(&paths.logs).join("cybercmd{}.log")
```

Reply:
That reminds me, I 'borrowed' that code from the documentation and should attribute it. Thus the different coding style. I'll fix that... if we keep that code.

I intend to switch the logger to `flexi_logger`.

I found another rust-based project called [redscript](https://github.com/jac3km4/redscript) in the CP2077 modding ecosystem by someone called "jekky" or "jac3km4". That project uses `flexi_logger` for logging. It outputs to the same log directory as the one I'm using. Assuming the logger is compatible with the asi dll, I thought it would be a better choice for consistency in both developer and user experience.

Especially since we are using the `log` facade that makes drop-in replacements trivial.

Reply:
I like `flexi_logger` and `fern`, but `log4rs` works too, I don't have a strong preference here

Reply:
Since neither of us has a strong preference for any logger, I copied `fn setup_logger(...)` from scc. It seemed like the easiest way to get a consistent experience. It also worked as a near drop-in replacement for my `fn setup_logging(...)`